### ISSUE TITLE
False positive `unused binding`

### ISSUE THREAD

**Issue by vemv** [clj-kondo/clj-kondo#1393]
**version**

09.25. Didn't happen with 09.15

**platform**

jvm

**problem**

FP `warning: unused binding sym`

**repro**

```clj
(defn- unmap-old-ns-symbols!
  "After renaming old-ns all symbols in the dependent ns ns-name still
  resolves to the wrong thing.  This unmaps those symbols so
  find-symbol doesn't get confused."
  [ns-name old-ns]
  (doseq [[sym _]
          (->> (ns-refers ns-name)
               (filter #(.startsWith (core/strip-reader-macros (val %))
                                     (str old-ns))))]
    (ns-unmap ns-name sym)))
```

Thanks - V

**Comment by borkdude** [comment]
Thanks! Fix one thing, get another bug in return :)

Will fix. For now you can use `#_:clj-kondo/ignore` before the unused sym.

**Comment by borkdude** [comment]
Minimal repro:
```
(doseq [sym ['foo 'bar 'baz]]
  (ns-unmap *ns* sym))
```

**Comment by borkdude** [comment]
Fixed on master.

**Comment by vemv** [comment]
Thanks as always ☕️
